packages/@ama-sdk/core/src/clients/api-fetch-client.ts

```diff
@@ -16,13 +16,19 @@ import {BaseApiClientOptions} from '../fwk/core/base-api-constructor';
 import {CanceledCallError, EmptyResponseError, ResponseJSONParseError} from '../fwk/errors';
 import {ReviverType} from '../fwk/Reviver';
 
-/** @see BaseApiClientOptions */
+/**
+ * @see BaseApiClientOptions
+ * @deprecated Use the one exposed by {@link @ama-sdk/client-fetch}, will be removed in v13
```

Comment:
Do we provide a ng-update or ng-add to handle that migration ?

Reply:
I added a rule in the `ng-add` of `@ama-sdk/client-fetch` to do the migration.